packages/allure-playwright/src/legacy.ts

```diff
@@ -44,105 +44,105 @@ export const allurePlaywrightLegacyApi: AllurePlaywrightLegacyApi = {
   /**
    * @deprecated please use import { label } from "allure-js-commons" instead.
    */
-  label: (...args) => allure.label(...args),
+  label: (...args) => Promise.resolve(allure.label(...args)),
```

Comment:
We don't need to wrap the methods into `Promise.resolve` because the wrapped methods return a promise or undefined, so we can add `await` keyword for both versions and nothing will be broken.

Reply:
`Promise.resolve` is required to convert `PromiseLike` to `Promise`